### PR TITLE
timml recipe added

### PR DESCRIPTION
**Pull request by mbakker7**
Added a recipe for timml. It includes a FORTRAN extension. I think it is ready to go. Let me know if not.

### CODE REVIEW

File: recipes/timml/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
+# Jinja variables help maintain the recipe as you'll update the version only here
```
Comment (synapticarbors) [inline recipes/timml/meta.yaml]: Please delete all comments.

File: recipes/timml/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+{% set name = "timml" %}
+{% set version = "5.0.0" %}
+
+package:
+   name: {{ name|lower }}
+   version: {{ version }}
+
+source:
```
Comment (synapticarbors) [inline recipes/timml/meta.yaml]: Is there a tarball available? See:

https://conda-forge.org/docs/meta.html#build-from-tarballs-not-repos

File: recipes/timml/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+  git_rev: v{{ version }}
+  git_url: https://github.com/mbakker7/timml.git
+
+build:
+  number: 0
+  skip: True  # [py<34]
+  string: py36_0
```
Comment (synapticarbors) [inline recipes/timml/meta.yaml]: I don't think we want to set the `string`

File: recipes/timml/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
```
Comment (synapticarbors) [inline recipes/timml/meta.yaml]: Please add `toolchain` to the `build` requirements

File: recipes/timml/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+
+source:
+  git_rev: v{{ version }}
+  git_url: https://github.com/mbakker7/timml.git
+
+build:
+  number: 0
+  skip: True  # [py<34]
+  string: py36_0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
```
Comment (synapticarbors) [inline recipes/timml/meta.yaml]: It doesn't look like the `setup.py` uses `setuptools`, so `--single-version-externally-managed` isn't going to be available.

File: recipes/timml/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+   - numpy x.x
+   - matplotlib >=1.4
+   
+test:
+  imports:
+  - timml
+  
+about:
+  home: https://github.com/mbakker7/timml
+  license: MIT
```
Comment (synapticarbors) [inline recipes/timml/meta.yaml]: Can you add the `license_file`, as well as `dev_url: https://github.com/mbakker7/timml`

File: recipes/timml/meta.yaml (hunk 1 above)
Comment (mbakker7) [inline recipes/timml/meta.yaml]: fixed

File: recipes/timml/meta.yaml (hunk 2 above)
Comment (mbakker7) [inline recipes/timml/meta.yaml]: gotta figure out how to do that

File: recipes/timml/meta.yaml (hunk 4 above)
Comment (mbakker7) [inline recipes/timml/meta.yaml]: added

File: recipes/timml/meta.yaml (hunk 6 above)
Comment (mbakker7) [inline recipes/timml/meta.yaml]: I tried to fix all the errors. No success yet. The sha256 is of the tar file not the tar.gz file. Does that matter?

**Comment by mbakker7** [comment]
I tried to fix all the errors. No success yet. The sha256 is of the tar file not the tar.gz file. Does that matter?

**Comment by mbakker7** [comment]
circleci reports a SHA256 mismatch, but I downloaded the intended file from github and used openssl sha256 to determine the checksum. Any idea why circleci gets a different checksum?

**Comment by mbakker7** [comment]
So far so good, but now circleci complains:

error: extension 'timml.besselaesnew' has Fortran sources but no Fortran compiler found

Does conda-forge not compile FORTRAN code? Do I need to upload something else than the FORTRAN code?

**Comment by jentjr** [comment]
@mbakker7 Mark, it looks like the circleci build couldn't find 'run_docker_build.sh'. Could you try to merge the upstream conda-forge/staged-recipes directory? I think the file structure has changed a bit.

**Comment by mbakker7** [comment]
Don't you think it may be easier to close the current PR and start a new
one from scratch? You already wrote the YML files.

**Comment by jentjr** [comment]
Yeah, that would be easier. If you want to close this PR, I can create a new one. Thanks.

**Comment by jentjr** [comment]
Do you want to tag a new version and I can create a new sha256 checksum?

**Comment by mbakker7** [comment]
We will start over

**Comment by mbakker7** [comment]
Just tagged a new release of timml

**Pull request closed without merging**